roles/openshift_master/defaults/main.yml

```diff
@@ -1,4 +1,9 @@
 ---
+# openshift_master_defaults_in_use is a workaround to detect if we are consuming
+# the plays from the role or outside of the role.
+openshift_master_defaults_in_use: True
```

Comment:
Can you re-explaining the meaning of the variable? "if we are consuming the plays from the role" is confusing

Reply:
During upgrades, systemd_units.yml is included in a playbook outside the role.  This means that the play is unable to consume the role's defaults/main.yml and vars/main.yml.  We have to work around by setting facts for variables that would otherwise be defined.  I'm using this because if 'openshift_master_defaults_in_use' is not defined, then that means someone is running the upgrade and the values in defaults/main.yml won't be detected.